IMD and IMD reset sections (2.2 and 2.5)

nrivkin opened the pull request.

Commit:
Added the requested test file for practice with pull requests

Commit:
Added a dream

Commit:
Added in start of IMD section 5.2 The Information in the table is correct, but unformatted. Only part of 5.2.1 has been written.

Commit:
esf main is updated to include the outline. The IMD properties table is formatted, but not labeled.

Commit:
Added caption to table. Rephrased IMD section 2.2.1

Commit:
Began work on section 2.2.3, position in car. need to figure out which section contains CAD for the accumulator and link to there.

Commit:
Began work on 2.2.2, IMD connectors and wiring. I have some concerns about our plan to use the 28 AWG wire. I described the current plan, but I am unsure if the connectors on the IMD will accept crimps that are small enough for 28 gauge. The IMD documentation suggests that 24 gauge is the minimum. Probably need to discuss with electrical leadership (Alex or Riley).

Commit:
Improved formatting after looking at updated esf_main from the master fork. Added a little more detail to 2.2.3. Commented the instructions, which I should have done a long time ago.

Commit:
added comments aand improved a bit of formatting in preparation to merge with updated master fork.

Commit:
Merged with main fork and resolved merge conflicts, though there may still be a few details I need to fix (duplicate lines, ect.)

Commit:
Resolved duplicates from merge. Added IMD latch schematic as a figure in section 2.2.2

Commit:
Added more detailed outline for section 2.5. Added specific references in section 2.2.1 and 2.2.2.

Commit:
Added explaination of IMD latch board to section 2.2.2, and described likely configuration of connectors with accumulator interface board. This completes rough draft of section 2.2

Commit:
Worked on section 2.5.1 and 2.5.3 In 2.5.1 wrote IMD reset procedure. In 2.5.3 referenced CAD for accumulator, and explained reasoning for position.

Commit:
Section 2.5.3 first draft complete. Added comments re AIR control board. Section 2.5.2 now mentions ribbon cable as well as the Molex ultrafit.

aehoppe reviewed (changes requested):
We're actually using a TE Connectivity MICRO MATCH connector, I'd check the part number there.

Commit:
Changed connector type to MICRO MATCH and added part number (338068-6)

Commit:
Added mention of fuse on high voltage line to IMD in section 2.2.2, but need to know rating of fuse.

Commit:
Changed fuse rating in 2.2.2 to 1 amp, which is what we used last year. I still need confirmation this has not changed, but until I have other info I am assuming it has not changed.

nrivkin commented:
Fixed connector type and added mention of fuse

aehoppe committed:
Merge branch 'master' into master

aehoppe commented on esf_main.tex:
```diff
@@ -207,33 +207,74 @@ \subsubsection{Position in car}
+
+
+\subsubsection*{2.2.2 Wiring/cables/connectors/}
+The  IMD uses the TYCO-MICRO MATE-N-LOK 1 x 2-1445088-8 connector and it's mate. It connects to the accumulator interface board, which sends the signal to a through a 28 AWG ribbon cable to the IMD latch using a 6-pin TE Connectivity MICRO MATCH 338068-6 connector.The 28 AWG wire is used because the current draw of the IMD is 150mA, which allows high gauge wire to be used. There is a 1 amp fuse p ...
```
As per last year's ESF, please provide a \ref to a section you create in the appendix for the IMD datasheet with the important parameters (and a hyperlink to the full datasheet), as well as a \ref to a section in the appendix where the fuse information will go. I will fill in the fuse information later today.

nrivkin replied:
done

aehoppe commented on esf_main.tex:
```diff
@@ -291,18 +332,37 @@ \subsubsection{Position in car/mechanical fastening/mechanical connection}
-TODO
+%TODO
+
+The IMD latch will be connected to the ?? board via a 4-pin Molex LLC Ultrafit 1722861104 with 20 AWG wire, and the ?? board using a  6-pin TE Connectivity MICRO MATCH 338068-6 connector with 28 AWG ribbon cable. The 20 AWG wire is used for the shutdown circuit and to supply power and ground to the IMD latch. The ribbon cable carries signals from the IMD.%On the ?? board it will connect , or anot ...
```
What are these question marks for? Also we need to figure out fusing for those wires and current and temperature ratings. Please reference last year's ESF for the important information to put here.

nrivkin replied:
Are we using the wiring found here? 
Mouser Part #:
571-19711119
Manufacturer Part #:
1-971111-9

http://www.mouser.com/ProductDetail/TE-Connectivity/1-971111-9/?qs=tNpDSWKagOPkPLjfSGNYAA%3d%3d

Commit:
Added apendix section with IMD datasheet and hyperlink. Added \ref regarding fuses. Mentioned all current ratings for wires, and voltage + temperature rating where brand of wire was known.

Commit:
Added apendix section with IMD datasheet and hyperlink. Added \ref regarding fuses. Mentioned all current ratings for wires, and voltage + temperature rating where brand of wire was known.

nrivkin commented:
Added IMD datasheet, added refs, added what I could find about wire types. For 20 AWG (in the shutdown circuit) and 18 AWG (for power transmission) I do not know the exact wire we are using so I could not find voltage or temp ratings. I got current ratings from http://www.powerstream.com/Wire_Size.htm, and assuming our ribbon cable was the one found here, or at least had equivalent ratings.http://www.mouser.com/ProductDetail/TE-Connectivity/1-971111-9/?qs=tNpDSWKagOPkPLjfSGNYAA%3d%3d

Commit:
Resolved errors from merge.

aehoppe commented:
Please add that since the IMD will be in the accumulator, the wiring to it will be rated to at least 300V and at least 90C.

Commit:
Wrote that wires would be specced at least to minimum requirements(300 volts and 90 degrees celcius). removed highlighting on previous changes.

aehoppe commented:
Closed because it was merged via a different PR
